### ISSUE TITLE
[Task] [Blob Tag] Phase || - Implement filter blob by Tag

### ISSUE THREAD

**Issue by blueww** [Azure/Azurite#1873]
Implement Following API and the related SAS change:

https://github.com/Azure/Azurite/blob/a1527fc0ec75137de6982b8dd3ce60fd38f9504d/src/blob/handlers/ContainerHandler.ts#L366
https://github.com/Azure/Azurite/blob/a1527fc0ec75137de6982b8dd3ce60fd38f9504d/src/blob/handlers/ServiceHandler.ts#L356

https://learn.microsoft.com/en-us/azure/storage/blobs/storage-manage-find-blobs?tabs=azure-portal#finding-data-using-blob-index-tags
https://learn.microsoft.com/en-us/rest/api/storageservices/find-blobs-by-tags-container?tabs=azure-ad#constructing-a-search-expression

**Comment by baswanders** [comment]
Yes, please, for us Azurite is unusable, because we use tags to save and locate blobs.

**Comment by Isylwin** [comment]
I second @baswanders, we'd like to use tags to filter blobs and the fact that this is unsupported would mean we can no longer run local and automated tests using Azurite.

**Comment by gorillapower** [comment]
Is there an estimated we could expect this feature?

**Comment by aaronenberg-msft** [comment]
Hello, we also depend on this feature for our dev and integration testing. Would be great to get this prioritized.

**Comment by stawr93** [comment]
Hello, is there any updates on the issue?

**Comment by kylescudder** [comment]
As of 2025-05-05 API version, the support Matrix says that Blob Tags (preview) is supported. Is this to say that FindBlobsByTagsAsync has been implemented?

**Comment by blueww** [comment]
Yes, this is already implemented in the latest Azurite release.
I will close this task.
